src/components/Calculator.js

```diff
@@ -1,39 +1,33 @@
 import React from 'react';
-import './Calculator.css';
-
-function Pad() {
-  return (
-    <div className="main">
-      <ul className="calc">
-        <li className="screen"><input className="" type="text" placeholder="0" /></li>
-        <li className="pad same"><button type="submit">AC</button></li>
-        <li className="pad same"><button type="submit">+/-</button></li>
-        <li className="pad same"><button type="submit">%</button></li>
-        <li className="pad different"><button type="submit">+</button></li>
-        <li className="pad same"><button type="submit">7</button></li>
-        <li className="pad same"><button type="submit">8</button></li>
-        <li className="pad same"><button type="submit">9</button></li>
-        <li className="pad different"><button type="submit">*</button></li>
-        <li className="pad same"><button type="submit">4</button></li>
-        <li className="pad same"><button type="submit">5</button></li>
-        <li className="pad same"><button type="submit">6</button></li>
-        <li className="pad different"><button type="submit">-</button></li>
-        <li className="pad same"><button type="submit">1</button></li>
-        <li className="pad same"><button type="submit">2</button></li>
-        <li className="pad same"><button type="submit">3</button></li>
-        <li className="pad different"><button type="submit">/</button></li>
-        <li className="pad same zero"><button type="submit">0</button></li>
-        <li className="pad same"><button type="submit">.</button></li>
-        <li className="pad different"><button type="submit">=</button></li>
-      </ul>
-    </div>
-  );
-}
+import './calculator.css';
```

Comment:
- [ ] As pointed out by the previous code reviewer, please fix the error in the screenshoot below be importing your css file properly. The name of the file is `Calculator.css` not `calculator.css`. Please kindly fix that.

![react_error](https://user-images.githubusercontent.com/65852621/212496394-2ddf8a84-0746-42ae-bf3f-2f268a960576.PNG)

Reply:
fixed